fog/ingest/server/src/controller_state.rs

```diff
@@ -63,6 +80,9 @@ pub struct IngestControllerState {
     /// Whether the server is idling, actively scanning the blockchain and
     /// publishing reports, or retiring
     mode: IngestMode,
+    /// Whether the server is unintialized, partially initialized, or fully
+    /// initialized
+    status: IngestStatus,
```

Comment:
I guess I don't see the value in adding this to the controller state

Is anything going to proceed differently if it was partially or fully initialized?

If not then this is probably dead code?

Reply:
Currently no. But I think it's important to flag the state difference so that overseer can handle these cases differently in the future. It is also useful for debugging, since if a user were to attempt to activate a node, it would be useful to know that the node is only in a partially initialized state.

Reply:
> But I think it's important to flag the state difference so that overseer can handle these cases differently in the future.

I think we should punt on this, and add this state in the future when we've actually designed what the overseer might do. Otherwise this is just dead code which increases the complexity of the system and the maintenance of it.